content/en/news/releases/1.6.x/announcing-1.6.5/index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Announcing Istio 1.6.5
+linktitle: 1.6.5
+subtitle: Patch Release
+description: Istio 1.6.5 patch release.
+publishdate: 2020-07-08
+release: 1.6.5
+aliases:
+    - /news/announcing-1.6.5
+---
+
+This release contains bug fixes to improve robustness. This release note describes
+what’s different between Istio 1.6.5 and Istio 1.6.4.
+
+{{< relnote >}}
+
+## Changes
+
+- **Fixed** support to sort the pods by `creationTimestamp` during mixer lookup and return the latest pod.
```

Comment:
**Fixed** mixer lookup to always return the proper source name when performing lookups by IP when multiple pods had the same IP.

Reply:
return the proper source name when mixer performing lookups by IP when multiple pods had the same IP.

Reply:
return the proper source name when mixer performing lookup by IP when multiple pods had the same IP.